### DISCUSSION TITLE
adminLTE 4 release

### DISCUSSION THREAD

**Discussion by hasan-4** [ColorlibHQ/AdminLTE#3662] [General]
when adminLTE 4 will be released ?

thank you

**Comment by danny007in** [comment]
Check https://github.com/ColorlibHQ/AdminLTE/projects/13
after 4 months

**Reply by hasan-4** [reply]
thank you

**Reply by ZhangChengLin** [reply]
> Check https://github.com/ColorlibHQ/AdminLTE/projects/13
> after 4 months

Will the official version be released in four months?
